Objects/unicodeobject.c

```diff
@@ -420,9 +420,10 @@ static void clear_global_interned_strings(void)
         return unicode_get_empty();  \
     } while (0)
 
-static inline void
-unicode_fill(int kind, void *data, Py_UCS4 value,
-             Py_ssize_t start, Py_ssize_t length)
+
+void
+_PyUnicode_Fill(int kind, void *data, Py_UCS4 value,
+                Py_ssize_t start, Py_ssize_t length)
```

Comment:
Are you sure that it will be inlined? Maybe keep static inline `unicode_fill()` and add a wrapper `_PyUnicode_Fill()`?

Reply:
> Maybe keep static inline unicode_fill() and add a wrapper _PyUnicode_Fill()?

Ok, I implemented _PyUnicode_Fill() as a static inline function. So the compiler can decide to inline it or not.